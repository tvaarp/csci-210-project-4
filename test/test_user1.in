sendmsg user2 hello there
sendmsg user2 see you later
