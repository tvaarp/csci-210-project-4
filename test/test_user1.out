Incoming message from user2: nice to meet you!
Incoming message from user2: bye
Exiting....
